src/FixedPointNumbers.jl

```diff
@@ -94,4 +94,17 @@ scaledual{T<:FixedPoint}(Tdual::Type, x::Union{T,AbstractArray{T}}) =
 scaledual{Tdual<:Number, T<:FixedPoint}(b::Tdual, x::Union{T,AbstractArray{T}}) =
     convert(Tdual, b/one(T)), reinterpret(rawtype(T), x)
 
+# printing
+@generated function show{T,f}(io::IO, x::FixedPoint{T,f})
```

Comment:
Right, this should not be `@generated`, it's not performance-critical and in any case being `@generated` won't help here.